test/e2e/integration/utils.go

```diff
@@ -25,18 +26,25 @@ type Cmd struct {
 
 // StopProcess is used to signal SIGTERM to the running process.
 func (cmd *Cmd) StopProcess() {
-	cmd.process.Process.Signal(os.Interrupt)
+	if err := cmd.process.Process.Signal(os.Interrupt); err != nil {
+		cmd.Logger.Errorf("failed to stop process %s: %v", cmd.Task, err)
+	}
 }
 
 // RunCmdWithFlags creates a process out of the  Cmd object.
 func (cmd *Cmd) RunCmdWithFlags() error {
-	cmd.process = exec.Command(cmd.Task, cmd.Flags...)
+	var err error
```

Comment:
this var declaration is not required if we create local variables with their scope belongs to their block.

Reply:
I just created it to avoid redefining it later if we remove/change the function call that first creates the `err` variable.

Reply:
yeah, but if look at the code, some part of code is re-declaring it, where you have made changes, you are not redeclaring it, can we have uniformity across this func ?

Reply:
Sure, I'll remove this and make it uniform then